### PR TITLE
Workshop: Introduction to Lists

### PR DESCRIPTION
**Pull request by brianegan**
My next workshop in the series, this one is titled "Introduction to Lists of Content." Where the Slivers workshop is aimed at Intermediate Flutter devs, this workshop aims to teach the basics of using ListView & GridView to beginners.

## Pre-launch Checklist

- [X] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [X] I signed the [CLA].
- [X] I updated/added relevant documentation (doc comments with `///`).
- [X] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-devrel channel on [Discord].

[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Comment by domesticmouse** [comment]
Do you have this staged somewhere so I can review it for the user's perspective?

File: dartpad_codelabs/src/introduction_to_lists/long_lists/snippet.dart [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// ignore_for_file: prefer_const_constructors, use_key_in_widget_constructors, prefer_const_literals_to_create_immutables
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(
+    MaterialApp(
+      home: Scaffold(
+        // Convert the normal ListView into a lazy ListView.builder
+        body: ListView(
```
Comment (domesticmouse) [inline dartpad_codelabs/src/introduction_to_lists/long_lists/snippet.dart:9]: nit: Adding a strategic `padding: EdgeInsets.all(8),` into the `ListView` declaraction can make for a much nicer looking result

Comment (brianegan) [reply dartpad_codelabs/src/introduction_to_lists/long_lists/snippet.dart:9]: Good call. I'll add a note about that in the first step!

Comment (brianegan) [reply dartpad_codelabs/src/introduction_to_lists/long_lists/snippet.dart:9]: Nice addition for sure, and it's a good one to teach folks early on. Introduced the padding and added it to a few steps.

**Review by domesticmouse — APPROVED** [review]

**Comment by domesticmouse** [comment]
Thanks @brianegan!

**Pull request merged**